aten/src/ATen/Declarations.cwrap

```diff
@@ -2220,6 +2221,15 @@
         - arg: THTensor* self
           broadcast: exponent fallback
         - THTensor* exponent
+]]
+[[
+  name: pow
+  types:
+    - floating_point
+  variants:
+    - function
```

Comment:
does it not work if this is a method?  (the existing pow is also a method).  Is this why it is separate from the existing pow declaration?

Reply:
This declaration (tpow) doesn't make sense as a method, since it's in the form `2^x`. The first argument is always a float.

The existing pow method doesn't treat this overload as a method.

When it was combined, the function_wrapper.py just ignored this overload because the signature looked like a duplicate.